ruby/cloudbuild-release.yaml

```diff
@@ -19,7 +19,7 @@ steps:
 - id: 'build-release'
   name: 'gcr.io/cloud-builders/docker'
   args: ['build', '-t', 'us-central1-docker.pkg.dev/$PROJECT_ID/release-images/yoshi-ruby/release', '.']
```

Comment:
Since these are new images and we will migrate their usage, can we rename the image to be `ruby-multi`?